package.json

```diff
@@ -53,11 +53,6 @@
 					"default": true,
 					"description": "Allows scan imported files."
 				},
-				"scss.scanImportedFilesDepth": {
```

Comment:
I like removal of this option. It doesn't make much sense to me.

You probably want to update README as well.